packages/kubernetes-1.15/kubelet-exec-start-conf

```diff
@@ -0,0 +1,24 @@
+[Service]
+ExecStart=
+ExecStart=/usr/bin/kubelet \
+{{~#unless settings.kubernetes.standalone-mode}}
```

Comment:
Any reason not to use an `if` here vs. an `unless/else`?

Reply:
At first I wasn't conditionalizing the `--cloud-provider` argument, so there was no `else` clause and I thought it read better.

I kept it since I expected the top branch to continue expanding while the bottom branch should always just be turning off the cloud provider, and it can go away in 1.23 when support for cloud providers in kubelet goes away.

Reply:
Ah, ok - seems good!